### ISSUE TITLE
Error 

### ISSUE THREAD

**Issue by LittlePotatoMan** [music-discussion/TownOfHost-TheOtherRoles#122]
Getting an error every time i try and install new update 

![image](https://user-images.githubusercontent.com/64777998/200075166-75c1c7a5-0bf3-48d8-9330-ecd0025ca925.png)

**Comment by Gurge44** [comment]
Launch Among Us through Steam/Epic, not the among us.exe in the folder.

**Comment by Gurge44** [comment]
Also, if this was with v0.9.3.3 and Steam, go back to a previous version, because the release was removed due to this exact problem. It should get fixed later in the day.

**Comment by GalaxyGamesJay** [comment]
No, this isn't why .9.3.3 was removed. Just create a text file in your mod folder called steam_appid.txt and paste the number below in it and save it.

945360
